Neos.TimeableNodeVisibility/Classes/Command/TimeableNodeVisibilityCommandController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Neos\TimeableNodeVisibility\Command;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Cli\CommandController;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryId;
+use Neos\TimeableNodeVisibility\Service\TimeableNodeVisibilityService;
+use Neos\TimeableNodeVisibility\Domain\HandlingResult;
+
+#[Flow\Scope('singleton')]
```

Comment:
I don't tihnk we need singleton for command controllers really, it was good practice at some point but I really don#t know why and don#t tihnk it's applicable anymore.

```suggestion
```